README.md

```diff
@@ -84,6 +84,7 @@ docker build -t stackstorm/st2notifier:1.1.2 st2notifier/
 docker build -t stackstorm/st2resultstracker:1.1.2 st2resultstracker/
 docker build -t stackstorm/st2rulesengine:1.1.2 st2rulesengine/
 docker build -t stackstorm/st2sensorcontainer:1.1.2 st2sensorcontainer/
+docker build -t stackstorm/st2garbagecollector:1.1.2 st2garbagecollector/
```

Comment:
And push:
https://github.com/StackStorm/st2-dockerfiles/blob/new_garbage_collector_service/README.md#4-deploy-to-docker-hub